include/crm_internal.h

```diff
@@ -375,4 +375,6 @@ void remote_proxy_end_session(const char *session);
 void remote_proxy_free(gpointer data);
 int  remote_proxy_check(lrmd_t * lrmd, GHashTable *hash);
 
+char* versioned_param_summary_for_digest(xmlNode *versioned_params, const char *name);
+void handle_versioned_parameters(xmlNode *param_set, xmlNode *versioned_params);
```

Comment:
Even though these are not part of the public API, they are exposed by libcrmcommon and thus could conflict with user code -- so let's prefix them with "crm_". Maybe "crm_versioned_param_summary()" and "crm_summarize_versioned_params()". I think it would make sense to put both of them in digest.c, since their only use is for calculating digests.